modules/dependencies/install.sh

```diff
@@ -6,8 +6,5 @@ ADDED_DIR=$(dirname $0)/added
 # Override default java.config file so that tls is not disabled
 cp $ADDED_DIR/java.config /etc/crypto-policies/back-ends/java.config
 
-# Remove dnf and rpm files as package manager not required at runtime
-rm -rf /var/lib/rpm /var/lib/dnf
```

Comment:
As this is only needed downstream, it might be worth us creating two modules for installing dependencies. A module to perform common installation, e.g. `ln -s ...` and then another which just performs upstream specific optimisations such as the one above.

The `dg-override.yaml` file would then specify only the modules required for downstream. WDYT?